extension/src/popup/views/AccountCreator/styles.scss

```diff
@@ -0,0 +1,40 @@
+.AccountCreator {
+  margin-bottom: 1rem;
+
+  &--header {
+    font-size: 2.125rem;
+    line-height: 1.2rem;
+    font-weight: normal;
+    color: var(--pal-text-primary);
+    text-align: center;
+  }
+
+  &--buttow-row {
+    padding: 1.5rem;
+  }
+
+  &--screen {
+    display: flex;
+    flex-flow: column wrap;
+    align-content: center;
+    justify-content: center;
+    // TODO - use HEADER_HEIGHT constant
+    height: calc(100vh - 119px);
+    max-height: 40rem;
+    max-width: 57rem;
+    width: 100%;
+    margin: auto;
+  }
+
+  &--half-screen {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0;
+    padding-top: 2rem;
+    padding-right: 0;
+    padding-bottom: 2rem;
+    padding-left: 1.55rem;
```

Comment:
This can be written shorthand as `padding: 2rem 0 2rem 1.55rem`;

Reply:
da2065e